src/poliastro/contrib/extract_czml.py

```diff
@@ -0,0 +1,288 @@
+import json
+
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import CartesianRepresentation
+from astropy.time import Time
+
+from poliastro.contrib.czml_extract_default_params import DEFAULTS
+
+
+class ExtractorCZML:
+    """A class for extracting orbitary data to Cesium"""
+
+    def __init__(self, orbit, start_epoch, end_epoch, N, fixed_rf=True):
+        """
+        Orbital constructor
+
+        Parameters
+        ----------
+        orbit: poliastro.Orbit
+            Orbit to be extracted
+        start_epoch: ~astropy.time.core.Time
+            Starting epoch
+        end_epoch: ~astropy.time.core.Time
+            Ending epoch
+        N: int
+            Number of sample points
+        fixed_rf: bool
+            Determines whether the reference frame is fixed
+            When set to false, it assumes the frame moving
+            with constant velocity.
+        """
+        self.czml = dict()
+        self.orbits = {1: [orbit, N, orbit.period, orbit.epoch]}
```

Comment:
Why do we need to store the period and epoch of the orbit? We have the `orbit` object itself, after all.

Reply:
Also, do we need `orbits` to be a dictionary? Or can it be a list?

Reply:
The problem with the current version of the code is that it has many leftover scribbles for things I'd like to implement next. 
I had this idea of adjusting the epoch of the timeline to the period of the orbit. As for storing the period, we don't need it but I think it would be a bit more readable than calling ``orbits[0].period`` then again... maybe not.

I used a dictionary for defining ``orbits`` as a way to quickly ID each orbit. The same thing could theoretically work with a list IF we decide on not deleting objects. Otherwise, using a list would be just too inefficient.

Reply:
> it has many leftover scribbles for things I'd like to implement next.

KISS and YAGNI!

https://www.wikiwand.com/en/KISS_principle
https://www.wikiwand.com/en/You_aren%27t_gonna_need_it

If it's not needed now, let's dump it :) There's always time to add it later when we realize "oh, now I need this kind of functionality, and the current implementation is limiting me". Let's strive for a simple implementation now **with good test coverage**: that will allow us to refactor later and assure that we are not breaking present functionality.

I prefer `orbits[0].period` than troublesome redundancy ("does this period _really_ correspond to this orbit?"). And yeah, I think we can remove the `del_orbit` functionality.

Reply:
Ah yes... I'm afraid I'm guilty of somewhat neglecting both practices... Anyway, thanks for the insightful comments. I'll start working on the changes as soon as I get back home!